### PR TITLE
uFrame Release Notes Version uframe_ooi_20161027_f6f2d988a971370b0504652023e7e3b90e5ad651

### PR DESCRIPTION
**Pull request by DanielJMaher**
Created from build Jenkins Deployment #146 and #147 feeding Jenkins Nightly Package #142

Do not merge yet.  Changes incoming.  Just creating this to help Bart.

### CODE REVIEW

**Comment by DanielJMaher** [comment]
@bchamilton Very close.  2 more things. You need a comma after every line the in the list ( list is everything in between these [ ] ).  Also you need quotes around Jenkins Nightly Package #142

File: ooiui/static/json/uFramePatchNotes.json [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
+        "VersionDescription":["Jenkins Nightly Package #142",
+            "Issue #11370 Correct 2 dimensional calibration data row/col order.  Remove commented code.",
+            "Issue #1006 Update Preload submodule",
+            "Issue #11199 Initial code to implement Preload database in Postgres.  Added new tables to Preload; updated existing Preload tables from the newest Preload.  Updated table column names with new preload model classes.",
+            "Issue #11412 Remove unused logback.xml configuration entries.",
+            "Issue #11407 - Catch Errors on AsyncResponse sent to user and correct. Reformat Address comments.",
+            "Issue #11413 UFrame now passes the fully qualified path name to stream engine for async requests.",
+            "Issue #11222 Add query for deployments associated with assets.  Fixed typos.",
+            "Issue #11455 Fixed Annotation column length that was not sufficient.",
+            "Issue #11431 Correct asset discovery in deployment population.  Corrected comments in code.  Add release notes.",
```
Comment (DanielJMaher) [inline ooiui/static/json/uFramePatchNotes.json]: @bchamilton Sorry, one last thing (i keep forgetting that JSON is new to you).  The last line in a list can't have a trailing comma.

File: ooiui/static/json/uFramePatchNotes.json [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
+            "Issue #1006 Update Preload submodule",
+            "Issue #11199 Initial code to implement Preload database in Postgres.  Added new tables to Preload; updated existing Preload tables from the newest Preload.  Updated table column names with new preload model classes.",
+            "Issue #11412 Remove unused logback.xml configuration entries.",
+            "Issue #11407 - Catch Errors on AsyncResponse sent to user and correct. Reformat Address comments.",
+            "Issue #11413 UFrame now passes the fully qualified path name to stream engine for async requests.",
+            "Issue #11222 Add query for deployments associated with assets.  Fixed typos.",
+            "Issue #11455 Fixed Annotation column length that was not sufficient.",
+            "Issue #11431 Correct asset discovery in deployment population.  Corrected comments in code.  Add release notes."
                 ]
     },
```
Comment (DanielJMaher) [inline ooiui/static/json/uFramePatchNotes.json]: @bchamilton Dont kill me! One more here!!  I'm sorry!

**Comment by bchamilton** [comment]
Dan,

I can't kill you for one more change because, I am too dim to see the
change?  What am I missing??

Thanks,
Bart


Bart Hamilton
Sr Systems Engineer/Test Engineer
Ocean Observatories Initiative
Intelligence, Information & Services
Raytheon Company
Desk  +1 402-390-8904
FAX   +1 402-390-2902
BCHamilton@Raytheon.com

2120 S. 72nd St, Suite 800
Omaha, NE 68124 USA
www.raytheon.com



This message contains information that may be confidential and privileged.
Unless you are the addressee (or authorized to receive mail for the
addressee), you should not use, copy or disclose to anyone this message or
any information contained in this message. If you have received this
message in error, please so advise the sender by reply e-mail and delete
this

**Pull request merged**